### ISSUE TITLE
I've already enabled Virtualization capabilities in the BIOS, but it still prompts me to do it.

### ISSUE THREAD

**Issue by NEPTLIANG** [microsoft/Windows-Sandbox-Utilities#15]
```
ERROR: Please Enable Virtualization capabilities in your BIOS settings...
```
![image](https://user-images.githubusercontent.com/28354311/78667592-d26a3b80-790b-11ea-8569-6bbc6be8e4c0.png)

**Comment by sebassem** [comment]
Do you have VTx enabled from the BIOS ?

**Comment by scarycrow-dave** [comment]
I have VTx enabled in Bios, 

![VTenabled](https://user-images.githubusercontent.com/63300122/78679233-f734c380-78b7-11ea-960c-725d5957dda4.JPG)

**Comment by sebassem** [comment]
It appears that the check for BIOS is not returing consistent results across different models , can you please share the output of both commands below in an elevated PowerShell session ?
`Get-ComputerInfo -Property HyperVRequirementVirtualizationFirmwareEnabled,hypervisorpresent`
`(Get-WmiObject win32_computersystem).hypervisorpresent`

**Comment by scarycrow-dave** [comment]
PS C:\WINDOWS\system32> Get-ComputerInfo -Property HyperVRequirementVirtualizationFirmwareEnabled,hypervisorpresent     
HyperVRequirementVirtualizationFirmwareEnabled HyperVisorPresent
---------------------------------------------- -----------------
                                                            True

PS C:\WINDOWS\system32> (Get-WmiObject win32_computersystem).hypervisorpresent
True

**Comment by Dreamer451** [comment]
I have the same issue and yes, virtualization is enabled in my BIOS. The output of the first command:

- HyperVRequirementVirtualizationFirmwareEnabled = True
- HyperVisorPresent = False

The second command results in False. This is on an HP OMEN an188nr.

**Comment by sebassem** [comment]
@scarycrow-dave thanks for sharing the output , you should not get this message if the second output if true which is the case as you shared , one more check please to confirm that virtualization is correctly enabled on your machine . Please run the below to enable the sandbox feature and validate if it was enabled successfully or you got an error message

`Enable-WindowsOptionalFeature -FeatureName 'Containers-DisposableClientVM' -All -Online -NoRestart`

**Comment by scarycrow-dave** [comment]
PS C:\WINDOWS\system32> Enable-WindowsOptionalFeature -FeatureName 'Containers-DisposableClientVM' -All -Online -NoRestart                                                                                                                      

Path          :
Online        : True
RestartNeeded : False

**Comment by sebassem** [comment]
> PS C:\WINDOWS\system32> Enable-WindowsOptionalFeature -FeatureName 'Containers-DisposableClientVM' -All -Online -NoRestart
> 
> Path :
> Online : True
> RestartNeeded : False

Thank you , will submit a fix shortly

**Comment by rdjr71** [comment]
The script is currently checking to see if a Hypervisor is already present.  That will return True if you have Hyper-V already installed (so the computer supports virtualization).  But that does not tell you if your computer supports Virtualization when it returns False.  

If it returns false for hypervisor present, you'll need to check to make sure these are true so you know virtualization is supported.

(GWMI Win32_Processor).VirtualizationFirmwareEnabled
and
(GWMI Win32_Processor).SecondLevelAddressTranslationExtensions

**Comment by sebassem** [comment]
> The script is currently checking to see if a Hypervisor is already present. That will return True if you have Hyper-V already installed (so the computer supports virtualization). But that does not tell you if your computer supports Virtualization when it returns False.
> 
> If it returns false for hypervisor present, you'll need to check to make sure these are true so you know virtualization is supported.
> 
> (GWMI Win32_Processor).VirtualizationFirmwareEnabled
> and
> (GWMI Win32_Processor).SecondLevelAddressTranslationExtensions

Thank you , we are trying to check if virtualization is enabled or not ,VirtualizationFirmwareEnabled returns false if hypervisor is enabled

**Comment by rdjr71** [comment]
Yeah, the presence of a hypervisor changes what you get when you query the CPU.  I would check for hypervisor=false, if false then do the additional 2 checks to make sure they are true.

**Comment by brasmith-ms** [comment]
Merged into master fff6af8e10496c37ca29501f2de3cbdb4c834ca8

**Comment by scarycrow-dave** [comment]
Thank you that fixed it :)
